app/scripts/views/mixins/signin-mixin.js

```diff
@@ -47,12 +47,17 @@ define(function (require, exports, module) {
           this.logFlowEvent('attempt', 'signin');
 
           // Check to see if this user is in the token code experiment,
-          // if so, override the verification method to use email-2fa
+          // if so, override to use the correct verification method. `email-2fa` sends
+          // an email with the verification code and `email` sends a confirmation link.
           let verificationMethod;
-          if (this.getTokenCodeExperimentGroup && this.getTokenCodeExperimentGroup() === 'treatment') {
+          if (this.getTokenCodeExperimentGroup && this.getTokenCodeExperimentGroup() === 'treatment-code') {
             verificationMethod = VerificationMethods.EMAIL_2FA;
           }
 
+          if (this.getTokenCodeExperimentGroup && this.getTokenCodeExperimentGroup() === 'treatment-link') {
```

Comment:
As these two conditions compare the same string, is it worth bringing them together as a `switch` statement inside a single `if (this.getTokenCodeExperimentGroup) ...` condition?